ArduCopter/defines.h

```diff
@@ -94,7 +94,8 @@ enum tuning_func {
     TUNING_RATE_YAW_FF =                54, // body frame yaw rate controller FF term
     TUNING_RATE_MOT_YAW_HEADROOM =      55, // motors yaw headroom minimum
     TUNING_RATE_YAW_FILT =              56, // yaw rate input filter
-    TUNING_WINCH =                      57  // winch control (not actually a value to be tuned)
+    TUNING_WINCH =                      57, // winch control (not actually a value to be tuned)
+    TUNING_ARTIFICIAL_CLIPPING =        58
```

Comment:
Please add a comment, "accelerometer clipping for high vibration failsafe testing" or something like that

Reply:
@amilcarlucas, thanks for having a look.  This is one of the commits with "-- do not merge" in the title though so this change won't make it into master... it's only to allow easier testing of this change.